README.rst

```diff
@@ -334,7 +334,30 @@ Miscellaneous functions
     >>> public_pem = ecdsa_key['keyval']['public']
     >>> ecdsa_key2 = import_ecdsakey_from_pem(public_pem)
 
+Testing
+~~~~~~~~~~~~~~~
+
+Testing is done with **tox**, which can be installed with pip:
+::
+
+    $ pip install tox
+
+
+Secure Systems Library supports multiple versions of Python.
+
+For that reason, the project is tested against multiple virtual environments with different Python versions.
+
+If you run 
```

Comment:
I think we can put "*Secure Systems Library supports [...]. If you run*" (L346-L350) all into one paragraph without blank lines.

Also, please remove the trailing whitespace after run. :)